app/routes/events/components/EventEditor/index.js

```diff
@@ -353,11 +353,43 @@ function EventEditor({
                   </Tooltip>
                 </div>
               )}
+            {['NORMAL', 'INFINITE'].includes(event.eventStatusType) && (
+              /* Need a checkbox to separate absolute registration and unregistration deadline */
+              /* Want to make a similar field for absolut unregistration time */
+
+              /* It will not be possible to unregister after this*/
+              <Tooltip content="Separate frister for påmelding og avmelding - antall timer før arrangementet. Det vil ikke være mulig å melde seg av eller på etter de satte fristene">
+                <Field
+                  label="Separate frister"
```

Comment:
What about _"Separat avregistreringsfrist"_? In my opinion it would be a little bit more unambigous.